<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CADSimplifierGui::DlgGetNeighborFaces</class>
 <widget class="QWidget" name="CADSimplifierGui::DlgGetNeighborFaces">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1117</width>
    <height>805</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Get NeighborFaces</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Shape</string>
     </property>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Selected shape:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="shapeObject">
        <item>
         <property name="text">
          <string>No selection</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="parameterName">
     <property name="title">
      <string>Fillet Parameter</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="labelfillet">
        <property name="text">
         <string>Fillet type:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="filletType">
        <item>
         <property name="text">
          <string>Constant Radius</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Variable Radius</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QTreeView" name="treeView"/>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="labelRadius">
          <property name="text">
           <string>Radius:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Gui::QuantitySpinBox" name="filletStartRadius" native="true">
          <property name="unit" stdset="0">
           <string notr="true">mm</string>
          </property>
          <property name="singleStep" stdset="0">
           <double>0.100000000000000</double>
          </property>
          <property name="value" stdset="0">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Gui::QuantitySpinBox" name="filletEndRadius" native="true">
          <property name="unit" stdset="0">
           <string notr="true">mm</string>
          </property>
          <property name="singleStep" stdset="0">
           <double>0.100000000000000</double>
          </property>
          <property name="value" stdset="0">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Selection</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="lableRadiusRange">
             <property name="text">
              <string>Radius Range</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="minRadius">
             <property name="maximumSize">
              <size>
               <width>256</width>
               <height>51</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="linkSymbol">
             <property name="styleSheet">
              <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
             </property>
             <property name="text">
              <string>~</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="maxRadius">
             <property name="maximumSize">
              <size>
               <width>256</width>
               <height>51</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>298</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QRadioButton" name="selectEdges">
             <property name="text">
              <string>Select edges</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="selectFaces">
             <property name="text">
              <string>Select faces</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="selectAllButton">
             <property name="text">
              <string>All</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="selectNoneButton">
             <property name="text">
              <string>None</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="selectFitButton">
             <property name="text">
              <string>SelectFit</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>558</width>
               <height>17</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::QuantitySpinBox</class>
   <extends>QWidget</extends>
   <header>Gui/QuantitySpinBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>shapeObject</tabstop>
  <tabstop>filletType</tabstop>
  <tabstop>treeView</tabstop>
  <tabstop>filletStartRadius</tabstop>
  <tabstop>filletEndRadius</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
